windows-server-container-tools/Debug-ContainerHost/Debug-ContainerHost.ps1

```diff
@@ -46,6 +49,13 @@ Describe "Docker is installed" {
                         -RedirectStandardError err.txt `
                         -RedirectStandardOutput dockerversion.txt
             $filesToDump["docker version"] = "dockerversion.txt"
+            Start-Process -NoNewWindow `
+                        -Wait `
+                        -FilePath docker.exe `
+                        -ArgumentList "network ls" `
+                        -RedirectStandardError err.txt `
+                        -RedirectStandardOutput dockernetworks.txt
+            $filesToDump["docker network"] = "dockernetworks.txt"
```

Comment:
recommend changing this to "docker network ls" so that it describes the full command that ran